addon.xml

```diff
@@ -1,13 +1,13 @@
 ﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-	<addon id="script.skin.helper.service" name="Skin Helper Service" version="1.1.24" provider-name="marcelveldt">
-	<requires>
-		<import addon="xbmc.python" version="2.13.0"/>
-		<import addon="xbmc.addon" version="12.0.0"/>
+    <addon id="script.skin.helper.service" name="Skin Helper Service" version="1.1.25" provider-name="marcelveldt, wuff">
+    <requires>
+        <import addon="xbmc.python" version="2.13.0"/>
```

Comment:
Bump to v3.0.0

Reply:
removed python req completely to remain compatible with Kodi 18 & 19